06week/dodgeball.js

```diff
@@ -0,0 +1,202 @@
+const arrOfPeople = [
```

Comment:
I really like that you put comments through out. But the indentation could use a little more work. It is not strictly necessary for the code to run, but it would make it easier to read and follow along